docker_entrypoint.sh

```diff
@@ -1,3 +1,27 @@
 #!/bin/sh
 
-exec tini hello-world
+set -e
+
+ITCHYSATS_USER=itchysats
+ITCHYSATS_PASSWORD=$(yq e '.password' /root/start9/config.yaml)
+
+# Properties page for username and password
+echo "version: 2
+data:
+  ItchySats Username:
+    type: string
+    value: \"$ITCHYSATS_USER\"
+    description: The username for ItchySats
+    copyable: true
+    masked: false
+    qr: false
+  ItchySats Password:
+    type: string
+    value: \"$ITCHYSATS_PASSWORD\"
+    description: The password for ItchySats.
+    copyable: true
+    masked: true
+    qr: false
+" > /root/start9/stats.yaml
+
+exec itchysats --data-dir=/data --http-address=0.0.0.0:8000 --password=$ITCHYSATS_PASSWORD mainnet --electrum=electrs.embassy:50001
```

Comment:
❓ Can we support testnet too?

Reply:
I have not found a simple way to run embassy on `testnet`;  also did not find any documentation for that, so I assume it is not supported at the moment.